news/2022-09-03-new-featured-artist-nashimoto-ui.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "New Featured Artist: Nashimoto Ui"
+date: 2022-09-03 10:00:00 +0000
+---
+
+**Nashimoto Ui** seeps into our Featured Artist library.
+
+![](https://assets.ppy.sh/artists/298/header.jpg)
+
+If this is a familiar sight, you're already familiar with both pain and **Nashimoto Ui**.
+
+![](/wiki/shared/news/2022-09-03-new-featured-artist-nashimoto-ui/miss.gif)
+
+The mind behind [AaAaAaAAaAaAAa](https://osu.ppy.sh/beatmapsets/1017271) has officially joined osu! as of today, providing a solid **7** dark VOCALOID tracks for mappers to dig into. Pick up any of 'em from [**Nashimoto Ui**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/298)!
+
+See what the hype is about with this slow-mo view of [ciru](https://osu.ppy.sh/users/6114695)'s DT play on *AaAaAaAAaAaAAa*, then check out the rest of these tracks while you're here:
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://assets.ppy.sh/artists/298/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### AaAaAaAAaAaAAa
+
+Try out [this jumpy map](https://osu.ppy.sh/beatmapsets/1017271) hosted by [Sotarks](https://osu.ppy.sh/users/4452992) or [this reading map](https://osu.ppy.sh/beatmapsets/1159170) hosted by [Morrighan](https://osu.ppy.sh/users/12042090)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/298/Songs/Nashimoto%20Ui%20-%20AaAaAaAAaAaAAa.mp3" type="audio/mpeg">
+</audio>
+
+### Enkou Shoujo
+
+Play through [this marathon](https://osu.ppy.sh/beatmapsets/1078896) hosted by [Sylvarus](https://osu.ppy.sh/users/4505918)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/298/Songs/Nashimoto%20Ui%20-%20Enkou%20Shoujo.mp3" type="audio/mpeg">
+</audio>
+
+### Shinitagari
+
+Maybe [try mapping this one yourself](LINK)!
```

Comment:
i think this would be the one

```suggestion
Maybe [try mapping this one yourself](https://assets.ppy.sh/artists/298/Songs/Nashimoto%20Ui%20-%20Shinitagari.osz)!
```